zio-http/src/main/scala/zhttp/http/HttpData.scala

```diff
@@ -91,10 +144,26 @@ object HttpData {
     RandomAccessFile(() => new java.io.RandomAccessFile(file, "r"))
   }
 
-  private[zhttp] final case class Text(text: String, charset: Charset)                        extends HttpData
-  private[zhttp] final case class BinaryChunk(data: Chunk[Byte])                              extends HttpData
-  private[zhttp] final case class BinaryByteBuf(data: ByteBuf)                                extends HttpData
-  private[zhttp] final case class BinaryStream(stream: ZStream[Any, Throwable, ByteBuf])      extends HttpData
-  private[zhttp] final case class RandomAccessFile(unsafeGet: () => java.io.RandomAccessFile) extends HttpData
-  private[zhttp] case object Empty                                                            extends HttpData
+  private[zhttp] final case class UnsafeContent(content: HttpContent) extends AnyVal {
+    def isLast: Boolean = content.isInstanceOf[LastHttpContent]
+  }
+
+  private[zhttp] final case class UnsafeChannel(ctx: ChannelHandlerContext) extends AnyVal {
+
+    def write(content: HttpContent): Unit         = ctx.write(content): Unit
+    def writeAndFlush(content: HttpContent): Unit = ctx.writeAndFlush(content): Unit
+    def read(): Unit                              = ctx.read(): Unit
+    def close(): Unit                             = ctx.close(): Unit
+    def flush(): Unit                             = ctx.flush(): Unit
```

Comment:
```suggestion
    def read(): Unit                              = ctx.read(): Unit
```
I think for now we only need `read` Let's keep it that way.